chainer/distributions/multivariate_normal.py

```diff
@@ -0,0 +1,198 @@
+import chainer
+from chainer.backends import cuda
+from chainer import distribution
+from chainer.functions.array import broadcast
+from chainer.functions.array import expand_dims
+from chainer.functions.array import moveaxis
+from chainer.functions.array import repeat
+from chainer.functions.array import squeeze
+from chainer.functions.array import stack
+from chainer.functions.array import swapaxes
+from chainer.functions.array import where
+from chainer.functions.math import basic_math
+from chainer.functions.math import exponential
+from chainer.functions.math import matmul
+from chainer.functions.math import sum as sum_mod
+from chainer import utils
+from chainer.utils import type_check
+import math
+import numpy
```

Comment:
Could you fix the import order?



> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

https://www.python.org/dev/peps/pep-0008/